rvps/README.md

```diff
@@ -80,6 +80,17 @@ Run
 docker run -d -p 50003:50003 rvps --address 0.0.0.0:50003
 ```
 
+Or we can build RVPS as a podman image
```

Comment:
The podman flow is different from "RVPS docker image" since it expects the binary is already available. It would be good to make it clear here.,